README.md

```diff
@@ -1,30 +1,32 @@
-# Tutorials for the Cosmos SDK
-
-!["Greetings Cosmonauts"](./hellochain/space.png)
+---
+title: Cosmos SDK Tutorials
+description: Tutorials for the Cosmos SDK.
+---
 
-_[The code for these tutorials (and this site) can be found here on Github](https://www.github.com/cosmos/sdk-tutorials)_
-
-## Prerequisites
+# Tutorials for the Cosmos SDK
 
-After a quick [Development Environment Setup](SETUP.md), you're ready to select a tutorial from the list below.
+<div>
+  <img src="./hellochain/space.png" style="max-width: 100px;">
+</div>
 
 ## [Nameservice](./nameservice/tutorial/00-intro.md)
 
 The real deal, full-length tutorial in which you will build a fully functional naming service on a blockchain. This tutorial does not use the `starter` package, rather you will be configuring and composing a full application yourself.
 The nameservice tutorial also has translations and a slide deck.
 
-#### [Live Testnet](https://talkshop.name)
-
-#### [Slides](https://docs.google.com/presentation/d/1aCMAdkVY-gfgnGNPTygwVk3o68czPQ_VYfvdMy9Ek5Q/edit?usp=sharing)
+- [Live Testnet](https://talkshop.name)
```

Comment:
I actually took this down recently. I still have a snapshot of the chain but was paying for it out of my personal account along with a number of other cosmos related projects and had to do a reduction. Happy to hand over the snapshot if you'd like to start it up again persistently. I wanted to write a "how to launch a chain with Digital Ocean" tutorial that it would be useful for too.

Reply:
Hmm.. maybe we could update the content in the next version. I wanted to tweak the docs theme config so I can fix the bugs in this PR altogether https://github.com/cosmos/vuepress-theme-cosmos/pull/61